contracts/tokenomics/maker/Cargo.toml

```diff
@@ -27,6 +27,7 @@ cw20 = { version = "0.8.0" }
 cw-storage-plus = { version = "0.8.0", features = ["iterator"] }
 schemars = "0.8.1"
 astroport = { path = "../../../packages/astroport", default-features = false }
+astroport-governance = { git = "https://github.com/astroport-fi/astroport-governance.git", branch = "feat/escrow-fee-distributor-contract" }
```

Comment:
Need to change to master once done

Reply:
Looks like we don't need this anymore

Reply:
We still need this until https://github.com/astroport-fi/astroport-governance/pull/7 merged

Reply:
Fixed